src/main/kotlin/lotto/ui/InputView.kt

```diff
@@ -0,0 +1,22 @@
+package lotto.ui
+
+object InputView {
+    fun purchaseAmountInput(): Int {
+        println("Please enter the purchase amount.")
+        val purchaseAmount =
+            readln().toIntOrNull() ?: throw IllegalArgumentException("Invalid input")
+        return purchaseAmount
+    }
```

Comment:
how about this? 😉 
it removes the `purchaseAmount`, avoids memory allocation.

```suggestion
        return readln().toIntOrNull() ?: throw IllegalArgumentException("Invalid input")
    }
```

Reply:
It makes sense thanks. I’ll update it like this.